tools/rawtools/rawtools.xml

```diff
@@ -0,0 +1,232 @@
+<tool id="rawtools" name="Raw Tools" version="@RAWTOOLS_VERSION@.@TOOL_SUBVERSION@">
+    <description>
+        Perform scan data parsing, quantification and quality control analysis of Thermo Orbitrap raw mass spectrometer files.
+    </description>
+    <macros>
+       <token name="@RAWTOOLS_VERSION@">1.4.1</token>
+       <token name="@TOOL_SUBVERSION@">0</token>
+       <xml name="input_fasta">
+         <param format="fasta" name="input_database" type="data" label="Protein Database"
+             help="Select FASTA database from history"/>
+       </xml>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@RAWTOOLS_VERSION@">rawtools</requirement>
+      <!--  <repository changeset_revision="349af9298ff1" name="proteomics_datatypes" owner="iracooke" prior_installation_required="True" toolshed="https://testtoolshed.g2.bx.psu.edu" /> -->
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Job Failed" />
+    </stdio>
+    <command>
+<![CDATA[
+        #from datetime import datetime
+        #import json
+        #import os
+        #set $exp_str = "Galaxy_Experiment_%s" % datetime.now().strftime("%Y%m%d%H%M%s")
+        #set $samp_str = "Sample_%s" % datetime.now().strftime("%Y%m%d%H%M%s")
+        #set $temp_stderr = "rawtools_stderr"
+        #set $bin_dir = "bin"
+
+        mkdir output;
+        cwd=`pwd`;
+        export HOME=\$cwd;
```

Comment:
If you add profile="18.01" this should work without this hack.
https://github.com/galaxyproject/galaxy/pull/5193

Reply:
We are very interested on keeping support for Galaxy 17.09...

Reply:
Outch, this version does not even get security patches afaik. But ok :)